### ISSUE TITLE
Scripts for reproduce experiments

### ISSUE THREAD

**Issue by darleybarreto** [wsgharvey/ps-nogs#1]
Hi, first of all great work and really nice coding! Would you happen to have bash or python scripts with all setups to reproduce the results of the paper?

Thank you in advance.

**Comment by wsgharvey** [comment]
Hi, thanks for raising this issue! I've put together a couple of bash scripts that should make them easier to reproduce. First, I added a script to more easily download the CelebA-HQ dataset (`download-celebhq.sh`). Then I've added `train-hard-attention-celebhq.sh`, which should run the experiments on training hard attention with partial supervision, and display the metrics we report, on CelebA-HQ. I've put descriptions for how to use them in the README.

I haven't yet added a bash script to generate new near-optimal glimpse sequences (i.e. running the code in `ps-nogs/generating-near-optimal-glimpse-sequences/`), as this will take a bit longer to test. Also, since we provide the near-optimal glimpse sequences generated by this section (in `ps-nogs/optimal-sequences/`), it is possible to use PS-NOGS by running `train-hard-attention-celebhq.sh` without having to regenerate the near-optimal sequences. I will upload a script for regenerating the near-optimal sequences in the next few days though. I'll also add scripts for the experiments on CUB in the near future. 

Let me know if there's anything else I can do to help.
